string robot_id
---
MapExchange data